backend/schema_dump/2023-04-23-193141_create_plants/up.sql

```diff
@@ -0,0 +1,176 @@
+-- Your SQL goes here
+CREATE TYPE DECIDUOUS_OR_EVERGREEN AS ENUM ('deciduous', 'evergreen');
+CREATE TYPE EXTERNAL_SOURCE AS ENUM (
+    'practicalplants',
+    'permapeople',
+    'reinsaat'
+);
+CREATE TYPE FERTILITY AS ENUM ('self fertile', 'self sterile');
+CREATE TYPE FLOWER_TYPE AS ENUM (
+    'dioecious',
+    'monoecious',
+    'hermaphrodite'
+);
+CREATE TYPE GROWTH_RATE AS ENUM ('slow', 'moderate', 'vigorous');
+CREATE TYPE HERBACEOUS_OR_WOODY AS ENUM ('herbaceous', 'woody');
+CREATE TYPE LIFE_CYCLE AS ENUM ('annual', 'biennial', 'perennial');
+CREATE TYPE LIGHT_REQUIREMENT AS ENUM (
+    'Full sun',
+    'Partial sun/shade',
+    'Full shade'
+);
+CREATE TYPE NUTRITION_DEMAND AS ENUM (
+    'light feeder',
+    'moderate feeder',
+    'heavy feeder'
+);
+CREATE TYPE PROPAGATION_METHOD AS ENUM (
+    'Seed - direct sow',
+    'Seed - transplant',
+    'Division',
+    'Cuttings',
+    'Layering',
+    'Spores',
+    'Seed'
+);
+CREATE TYPE SHADE AS ENUM (
+    'no shade',
+    'light shade',
+    'partial shade',
+    'permanent shade',
+    'permanent deep shade'
+);
+CREATE TYPE SOIL_PH AS ENUM (
+    'very acid',
+    'acid',
+    'neutral',
+    'alkaline',
+    'very alkaline'
+);
+CREATE TYPE SOIL_TEXTURE AS ENUM (
+    'sandy',
+    'loamy',
+    'clay',
+    'heavy clay'
+);
+CREATE TYPE SOIL_WATER_RETENTION AS ENUM ('well drained', 'moist', 'wet');
+CREATE TYPE WATER_REQUIREMENT AS ENUM ('Dry', 'Moist', 'Wet', 'Water');
+CREATE TABLE plants (
+    id SERIAL PRIMARY KEY,
+    unique_name TEXT NOT NULL,
+    common_name_en TEXT [],
+    common_name_de TEXT [],
+    family TEXT,
+    genus TEXT,
+    edible_uses_en TEXT,
+    medicinal_uses TEXT,
+    material_uses_and_functions TEXT,
+    botanic TEXT,
+    material_uses TEXT,
+    functions TEXT,
+    heat_zone SMALLINT,
+    shade TEXT,
+    soil_ph SOIL_PH [],
+    soil_texture SOIL_TEXTURE [],
+    soil_water_retention SOIL_WATER_RETENTION [],
+    environmental_tolerances TEXT [],
+    native_geographical_range TEXT,
+    native_environment TEXT,
+    ecosystem_niche TEXT,
+    deciduous_or_evergreen DECIDUOUS_OR_EVERGREEN,
+    herbaceous_or_woody HERBACEOUS_OR_WOODY,
+    life_cycle LIFE_CYCLE [],
+    growth_rate GROWTH_RATE [],
+    height TEXT,
+    width TEXT,
+    fertility FERTILITY [],
+    flower_colour TEXT,
+    flower_type FLOWER_TYPE,
+    created_at TIMESTAMP DEFAULT now() NOT NULL,
+    updated_at TIMESTAMP DEFAULT now() NOT NULL,
+    has_drought_tolerance BOOLEAN,
+    tolerates_wind BOOLEAN,
+    plant_references TEXT [],
```

Comment:
Better to not repeat the table name:

```suggestion
    references TEXT [],
```